casper/src/main/scala/coop/rchain/casper/engine/LfsTupleSpaceRequester.scala

```diff
@@ -149,83 +160,79 @@ object LfsTupleSpaceRequester {
               .whenA(isReceived)
 
         // Import chunk to RSpace
-        _ <- Stream
-              .eval {
-                // Transform values from ByteString to ByteVector
-                val historyItemsBytes = historyItems.map {
-                  case (k, v) => (k, ByteVector(v.toByteArray))
-                }
-                val dataItemsBytes = dataItems.map {
-                  case (k, v) => (k, ByteVector(v.toByteArray))
-                }
-
-                // Validation for received tuple space items.
-                val validationProcess = Stream.eval {
-                  Stopwatch.time(Log[F].info(_))("Validate received state items")(
-                    validateTupleSpaceItems(
-                      historyItemsBytes,
-                      dataItemsBytes,
-                      startPath,
-                      chunkSize = pageSize,
-                      skip = 0,
-                      importer.getHistoryItem
-                    )
-                  )
-                }
+        _ <- (Sync[F].defer[Unit] _ andThen Stream.eval) {
```

Comment:
what does this syntax mean?

Reply:
It's Scala way for function composition. It means `x => Stream.eval( Sync[F].defer[Unit]( x ) )`.

```scala
f andThen g   ==   x => g(f(x))   ==   g compose f
```